.circleci/config.yml

```diff
@@ -50,7 +50,7 @@ commands:
           name: Install Dependencies
           command: |
             sudo apt-get update
-            sudo apt-get install -y cmake curl clang llvm
+            sudo apt-get install -y cmake curl clang llvm gcc-powerpc-linux-gnu
```

Comment:
do you actually need gcc-powerpc-linux-gnu here?

Reply:
Yes, something in the transitive dependencies needed gcc so I added it after the first attempts failed.